doc/programming_guide/quickstart.rst

```diff
@@ -190,9 +190,15 @@ The `examples/media_player.py` example demonstrates playback of streaming
 audio and video using pyglet.  The `examples/noisy/noisy.py` example
 demonstrates playing many short audio samples simultaneously, as in a game.
 
+.. note:: Linux users may want to install OpenAL.
```

Comment:
I would remove this part. It's a little too much information for the Quick Start guide IMO.